src/components/typography/typography.component.tsx

```diff
@@ -46,214 +53,80 @@ export interface TypographyProps extends SpaceProps, ColorProps {
   display?: string;
   /** Override the list-style-type */
   listStyleType?: string;
-  /** Override the white-space type */
+  /** Override the white-space */
   whiteSpace?: string;
-  /** Override the word-wrap type */
+  /** Override the word-wrap */
   wordWrap?: string;
-  /** Override the text-overflow type */
+  /** Override the text-overflow */
   textOverflow?: string;
   /** Apply truncation */
   truncate?: boolean;
+  /** Override the color style */
+  color?: string;
+  /** Override the backgroundColor style */
+  backgroundColor?: string;
+  /** Override the bg value shorthand for backgroundColor */
+  bg?: string;
+  /** Override the opacity value */
+  opacity?: string | number;
+  /** @private @ignore */
+  className?: string;
```

Comment:
**question** (non-blocking): same comment applies here, do we need to be adding `className`? Perhaps there's a requirement i'm not privvy too

Reply:
`className` is used and its current removal will break tests. Potentially we can remove `className` in another ticket though